doc/files.md

```diff
@@ -45,7 +45,7 @@ Chain option                   | Data directory path
 
 Subdirectory       | File(s)               | Description
 -------------------|-----------------------|------------
-`blocks/`          |                       | Blocks directory; can be specified by `-blocksdir` option (except for `blocks/index/`)
+`blocks/`          |                       | Blocks directory; can be changed by specifying the parent directory with the `-blocksdir` option (except for `blocks/index/`)
```

Comment:
> Core GUI menu option Help->Command Line Options, explains it properly

Yes, as does `./src/bitcoind -h | grep -A3 blocksdir`, which is the help documentation on how to use the -blocksdir option.

I think the vocation of `files.md` is to describe the role of each file/directory and not how to use configuration options, so  the help docs would be the right place to maintain this information without redundancy.

Reply:
Quite right that the true documentation is maintained elsewhere. But don't you agree that this doc is unclear? And if so then why not fix it? Or remove the `blocksdir` note altogether

Reply:
NACK

Though this doc may not contain much detail about the usage of `-blocksdir`, it's certainly not the best place to make usage clarifications; as @jonatack has also mentioned, the file's purpose isn't explicitly for config options usage info. 

If the user needs more clarity on the `-blocksdir` option, they can already check the usage either through `./src/bitcoind -h | grep -A3 blocksdir` or `Help->Command Line Options`, for example.

Reply:
I think that the proposed changed is more confusing. Someone reading this document will first read this
https://github.com/bitcoin/bitcoin/blob/1bad29fe02096b269858b192a4a91c785a44e505/doc/files.md?plain=1#L35

Which implicitly says that the blocks directory is chain-specific. Therefore the `blocksdir=` is indeed specifying the general/parent/all-chains blocks directory because it must somehow separate between chains, which does by creating chain-specific folders one level deeper.